python/tank_vendor/yaml/__init__.py

```diff
@@ -1,10 +1,20 @@
-from ..shotgun_api3.lib import six
```

Comment:
That import does not even make any sense. I'm shocked it worked in the first place some of the time, as we're importing something across packages.